src/s3b_the_roach_market.tw2

```diff
@@ -344,3 +357,46 @@ I’d found this same box on this same table as a kid, and snatched it up immedi
+
+We’re gonna head out to the far fields – where you can actually see the river. The only lights are a Fed boat doing a routine patrol and the only people dudes who’ve wandered too far on a bad trip.
+
+Leo comes roaring up on this tricked-out, midnight black scooter. It’s got pristine hatchet man decals and a fresh paint job. It’s so new that it seems like it’s sparkling all over.
+
+He pulls up next to me, the engine thundering so loud I have to scream at him. “LOOK AT YOU, HUH? [SOME BIG DEAL NOW]<scooterTrigger2|?”]]
+
+(click: ?scooterTrigger2)[$dissolve[I knew it was what he wanted, but I thought that would make the conversation end sooner. He sidles up even closer and cuts the engine. “Pop took a job for the Honchos. Fixed up a few old vans,” he grinned, “We’re rolling in it this week.”
+
+Before I can even respond, he says, “Wanna live like a queen today?”
+
+I know what he wants. We all know what boys want from now until the end of time. It’s not like I’m naive or that I don’t want some version of love too. It felt like every opportunity to say yes might be my last chance.
+
+So I said yes, more to finding out what was next than to him in particular, and we rode off together into the beginning of our story.
+
+I thought I’d see Anna with a big frown, watching me through the scooter dust. But, when I looked [she wasn’t there]<scooterTrigger3|.]]
+
+(click: ?scooterTrigger3)[$dissolve[When I found her the next day, she just wanted details. I barfed a flood of so much boring detail that it should’ve crushed us both. I started sketching his name on things in idle silences.
+
+By the next week, she was tired of hearing about us. I started practicing alone before shows and moved my baggage over to the new relationship. Took a few looks at the old friendship before I tossed the keys and [closed the door]<scooterTrigger4|.]]
+
+(click: ?scooterTrigger4)[$dissolve[When it ended with Leo three weeks later, I felt busted worse than a snowglobe hit with a baseball bat. It hadn’t taken long for his new ride to get dinged up and lose its shine. Some shithead scraped it for a laugh, later his buddies wanna take a joy ride and they took it into a ditch.
+
+It was the same with his personality – a tossed off nasty remark here, a selfish choice there. His good intentions peeled faster than the scooter’s cheap paint. [Nothing nice lasts in Gatherville]<scooterTrigger5|.]]
+
+(click: ?scooterTrigger5)[$dissolve[After we split, all the things I used to love, like (if: $isWrestler is true)[practicing my grapples](else:)[spitting rhymes over a tape loop], felt like they had their joy buried under an emotional goop.
+
+I couldn’t access them in the same way, even if I forced myself through the motions. Everything I tried to write, even diary entries, turned into whiny sludge. I had become a shot of snot at the rim of a sink drain.
```

Comment:
"access" strikes me as an odd word in this context. It's more about headspace, right?

"I couldn't get into the zone again, even when I forced myself to go through the motions" maybe?

Reply:
Changed "access" to "enjoy" to simplify.